site/admission/index.html

```diff
@@ -0,0 +1,121 @@
+  <p>Jodie McSally Dance Program, Class of 21</p>
+  <div>
+    <div>Overview</div>
+    <div>Application Process</div>
+    <div>Auditions</div>
+    <div>FAQ</div>
+  </div>
+  <div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+      magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut.</p>
+  </div>
+  <h5>Students must live in Boston by TKTK in order to enroll. Learn more</h5>
+  <div>
+    <div>Grades Blind</div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing
+      elit, sed do eiusmod tempor incididunt ut labore et
+      dolore magna aliqua. Ut enim ad minim.</p>
+  </div>
+  <div>
+    <div>All Levels of Experience</div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing
+      elit, sed do eiusmod tempor incididunt ut labore et
+      dolore magna aliqua. Ut enim ad minim.</p>
+  </div>
+  <div>
+    <div>Full Inclusion</div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing
+      elit, sed do eiusmod tempor incididunt ut labore et
+      dolore magna aliqua. Ut enim ad minim.</p>
+  </div>
+  <h3>Upcoming Admissions Events</h3>
+  <p>View All</p>
+  <div>
+    <div>Nov 10</div>
+    <p>Sign up for open house</p>
+  </div>
+  <div>
+    <div>Nov 10</div>
+    <p>Sign up for open house</p>
+  </div>
+  <div>
+    <div>Dec 17</div>
+    <p>Sign up for Audition Workshop</p>
+  </div>
+  <div>
+    <div>Jan 23</div>
+    <p>Sign up for Alumni Panel: Opportunity After BAA</p>
+  </div>
+  <h2>Application Process</h2>
+  <div>
+    <h1>1</h1>
+    <div>Register in BPS (lorem ipsom dolsot) | Due 1/22/21
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      Register in BPS (Lorem ipsom dolsot)
+    </div>
+  </div>
+  <div>
+    <h1>2</h1>
+    <div>BPS Choice Form | Due 1/22/21
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      Fill out Choice Form
+    </div>
+  </div>
+  <div>
+    <h1>3</h1>
+    <div>Apply to Boston Artst Academy | Rolling (earkuer us better)
+      Create an account on Accepted.com and submit your
+      application!
+      Submit Your Application to BAA
+    </div>
+  </div>
+  <div>
+    <h1>4</h1>
+    <div>Schedule an Audition | Due 2/15/21
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
+      Schedule an Audition
+    </div>
+  </div>
+  <div>
+    <h1>5</h1>
+    <div>Decisions Go Out | Check Your Email on 1/22/21
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore.
+      Attend An Admitted Students Event
+    </div>
+  </div>
+  <div>
+    <h1>6</h1>
+    <div>Enroll in BAA | Due TKTKTK
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+      do eiusmod tempor incididunt ut labore.
+      View Enrollment Process
+    </div>
+  </div>
+  <h2>Questions? We have answers! loremipsom@baa.org | 617-650-3434.</h2>
```

Comment:
I'd probably make this a `<p>` or a higher level header like an `<h3>` or `<h4>`. I don't see it as a header mostly because it doesn't introduce a section of the page, it's more just a large piece of text.